### PR TITLE
LPD-21742 - Replace empty state image GIF for SVG (3/3)

### PR DESCRIPTION
**Pull request by ilzamcmed**
Jira issue: https://liferay.atlassian.net/browse/LPD-21742

This PR is part one of three, to add empty state images and replace the GIF for SVG.

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ethib137** [comment]
Hey @marcoscv-work shouldn't using svg mean that we can have one svg that can be animated by default and not animated with reduced motion? In this PR we are using two svg's one for normal and one for reduced motion. Is that intended?

**Comment by matuzalemsteles** [comment]
> shouldn't using svg mean that we can have one svg that can be animated by default and not animated with reduced motion? In this PR we are using two svg's one for normal and one for reduced motion. Is that intended?

I discussed this with @ilzamcmed because this will not work because the information inside the img is isolated and as if it were an iframe, then changing the style in the main window will not impact the style inside the svg of the image being loaded, so for example the `c-prefers-reduced-motion` will not work and neither will media prefers-reduced-motion. I also did some tests with it to be sure.

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Review by pat270 — APPROVED** [review]

**Comment by pat270** [comment]
Adding to Matu's comment, if we inline the SVG, we can reuse the code.

**Comment by ethib137** [comment]
> I discussed this with @ilzamcmed because this will not work because the information inside the img is isolated and as if it were an iframe, then changing the style in the main window will not impact the style inside the svg of the image being loaded, so for example the `c-prefers-reduced-motion` will not work and neither will media prefers-reduced-motion. I also did some tests with it to be sure.

> Adding to Matu's comment, if we inline the SVG, we can reuse the code.

Okay, this is definitely a better reason to consider inlining the svg directly in the component. We would still need to include the svg's in the theme to handle the non-react instances in Liferay, but this would be a good improvement for instances that use the React component. 

So this PR looks okay, we just need to remove the changes to workspaces like in the other ones.

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
:warning: 
This pull request depends on changes introduced in the Clay Release. It can only be merged after the Clay Release is merged into the master branch.
See https://github.com/liferay-platform-experience/liferay-portal/pull/562

**Comment by ilzamcmed** [comment]
ci:test:relevant

File: modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagList.tsx [hunk 1]
```diff
@@ -39,8 +39,8 @@ const FeatureFlagList: React.FC<IFeatureFlagListProps> = ({featureFlags}) => {
 							'no-feature-flags-were-found'
 						),
 					}}
-					imgSrc={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/search_state.gif`}
-					imgSrcReducedMotion={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/empty_state_reduced_motion.gif`}
+					imgSrc={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/search_state.svg`}
+					imgSrcReducedMotion={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/empty_state_reduced_motion.svg`}
```
Comment (ethib137) [inline modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagList.tsx:43]: @ilzamcmed can you create a separate task to make sure we are using search_state_reduced_motion not empty_state_reduced_motion? Also, you will notice that in feature flags we never actually show the `empty_state_reduced_motion` even when reduced motion is on. We also need a new Clay ticket created to respect the `imgSrcReducedMotion` prop.

**Review by ethib137 — APPROVED** [review]

**Comment by ethib137** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:stable

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:forward:force

**Pull request closed without merging**